scripts/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+# ~~~
```

Comment:
Changes to this file improve performance quite a bit and uses the modern approach for finding python. It also has much better android support.